schemas/FeatureManagement.Dotnet.v2.0.0.schema.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I wouldn't consider this v2.0.0 of anything. This is separate

Reply:
I would say that this should be defined in the Azure/AppConfiguration repo where the Microsoft FeatureFlag schema is defined, and it should contain the root `FeatureManagement` node.

Reply:
PR sent out: https://github.com/Azure/AppConfiguration/pull/871